js/src/dapps/dappreg/Modal/modal.css

```diff
@@ -15,95 +15,120 @@
 /* along with Parity.  If not, see <http://www.gnu.org/licenses/>.
 */
 
+@import '../_colors.css';
+@import '../_utils.css';
+
 .modal {
-  .body {
-    bottom: 0;
-    left: 0;
-    position: fixed;
-    right: 0;
-    top: 0;
-    text-align: center;
-    z-index: 50;
+  align-items: center;
+  background-color: rgba(40, 40, 40, 0.75);
```

Comment:
Also would have put this in `colors.css` (There are a couple, not commenting on all). Not a crisis where there are no re-use, bust consistency for all.